### PR TITLE
SEN-1235 -- first post

### PR DESCRIPTION
**Pull request by noTxt**
**Reviewing:**
- checkout branch
- `bundle exec jekyll serve`
- [visit blog](http://localhost:4000/)

**Ticket:**
https://cascadeenergy.atlassian.net/browse/SEN-1235

### CODE REVIEW

**Comment by noTxt** [comment]
*note:*
Same as on https://github.com/CascadeEnergy/CascadeEnergy.github.io/pull/6, I think velocity is more important than perfection for this post

File: _drafts/called-out.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+During all that change we made a lot of progress. We adopted and codified better practices, quantified success, released updates more often. But we left one piece behind. Something from that original dream that drew us here in the first place.
+
+The occasional conversation about "thought leadership" sounded more and more far-fetched. Hollow. Unrealistic even. Eventually we stopped talking about it at all. And then we stopped thinking about it. We settled into our workstations, put on our headphones, and only broke the silence with our key strokes.
+
+Until last week.
+
+When [our CEO](https://www.linkedin.com/pub/marcus-wilcox/17/453/46B) confronted us with the question: "Do you feel that other groups at cascade understand what you do."† We squirmed uncomfortably in our seats, protested busyness, ducked and weaved responsibility to the best of our ability. With the patience of a father, he refuted all of our points until we were staring at an ugly truth. We'd stopped trying.
+
```
Comment (jinsyaoommen) [inline _drafts/called-out.md]: What would be a better word for `busyness`?

File: _drafts/called-out.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+We'd stopped trying to communicate why what we do is so *exciting*. 
+
+So this blog is a place where we can begin remedy that. For our company, our city, our peers in the craft. It's not the entire solution, but it's important. We want to tell the story of our daily efforts. The problems that we face and the victories we share. Whether it's a technical battle against DB IO or the personal struggle of getting to work on time, we want to be better about connecting our experiences with **yours**.
+
```
Comment (jinsyaoommen) [inline _drafts/called-out.md]: `we want to be **better**` ?

Comment (noTxt) [inline _drafts/called-out.md]: instead of "yours"?

**Pull request closed without merging**